src/backend/aspen/workflows/nextstrain_run/export_test.py

```diff
@@ -25,14 +25,48 @@
     type=click.Choice(["overview", "targeted", "non_contextualized"]),
     required=True,
 )
-@click.option("--sequences", type=int, required=False, default=10)
-@click.option("--selected", type=int, required=False, default=10)
-@click.option("--gisaid", type=int, required=False, default=10)
+@click.option(
+    "--sequences",
```

Comment:
small nit but i think `included-sequences` might be a better flag name

Reply:
or something to refer more intuitively that this means a number of sequences to include, idk maybe i'm the only one confused

Reply:
actually it's fine, ignore me